src/domain/callout/CalloutsPage.tsx

```diff
@@ -52,10 +51,22 @@ const CalloutsPage = ({ entityTypeName, rootUrl }: CalloutsPageProps) => {
                   hubNameId={hubNameId!}
                   challengeNameId={challengeNameId}
                   opportunityNameId={opportunityNameId}
+                  showCreateButton={canCreateCallout}
```

Comment:
Creating callouts and creating aspects are bound to different privileges.

Reply:
addressed in my last commit

Reply:
Merging to dev, to be re-verified after all-hands session